FHEM/14_SD_WS.pm

```diff
@@ -542,7 +544,8 @@ sub SD_WS_Parse($$)
 	    	$temp=$decodingSubs{$protocol}{temp}->( $rawData,$bitData );
 	    	$hum=$decodingSubs{$protocol}{hum}->( $rawData,$bitData );
 	    	$channel=$decodingSubs{$protocol}{channel}->( $rawData,$bitData );
-	    	$model = $decodingSubs{$protocol}{model};
+	    	$model = $decodingSubs{$protocol}{model};						# for models without Humidity
+			$model = $decodingSubs{$protocol}{model}."H" if $hum != 0;		# for models with Humidity
```

Comment:
Müsste hier nicht noch eine Einschränkung dazu, damit es das "H" nicht zusätzlich bei allen Sensoren mit Feuchtewert anhängt?

				if ($model eq "SD_WS_33_T") { 
					$model = $decodingSubs{$protocol}{model}."H" if $hum != 0;		# for models with Humidity
				}